horreum-web/src/domain/runs/components.tsx

```diff
@@ -160,14 +160,15 @@ function useUpdateDescription(run: RunSummary): MenuItem<RunSummary> {
     ]
 }
 
-export function Menu(run: RunSummary, refreshCallback: () => void) {
+export function Menu(run: RunSummary, refreshCallback: () => void, clearSelected: Function) {
```

Comment:
please define the `clearSelected` function in the same way as the `refreshCallback` function is defined, i.e. `clearSelected: () => void` that way it is clear what the expected args and and the return type

The function name should reflect that it is a callback as well, so maybe rename to `clearCallback`

Reply:
ok